src/commands/create/create.ts

```diff
@@ -46,15 +79,34 @@ export class CreateCommand implements CommandModule<GlobalArguments, CreateArgum
   };
 
   public handler = async (argv: Arguments<CreateArguments>): Promise<void> => {
-    const { s3BucketName, s3KeyId, s3ScriptKey, dumpFilePath, dumpServerUrl } = argv;
+    const { s3KeyId, s3ScriptKey, s3BucketName, dumpLocalFile, source } = argv;
     const scriptKey = join(s3KeyId, s3ScriptKey);
 
-    const scriptPath = await this.manager.getScriptFromS3ToFs(s3BucketName, scriptKey);
+    try {
+      const scriptPath = await this.manager.getScriptFromS3ToFs(s3BucketName, scriptKey);
+
+      let dumpPath = dumpLocalFile;
+      if (source === DumpSource.REMOTE_URL) {
+        dumpPath = await this.manager.getDumpFromRemoteToFs(this.dumpSource as string);
+      } else if (source === DumpSource.DUMP_SERVER) {
+        dumpPath = await this.manager.getFromDumpServerToFs(this.dumpSource as string);
+      }
```

Comment:
use same action for both options, just add a middle step

Reply:
done.